core/arch/arm/kernel/entry_a64.S

```diff
@@ -265,6 +265,13 @@ clear_nex_bss:
 	/* Enable Console */
 	bl	console_init
 
+#ifdef CFG_CORE_STACK_PROTECTOR
```

Comment:
This should probably be moved just before we're cleaning the cache to let platforms use higher level functions if possible. Same for the 32-bit code.

Reply:
I'm a bit confuse here. Not sure how moving it before the cache clean (line 253) would allow platforms to use higher level functions. Maybe you mean move it after we enable the MMU and re-initialize the serial console at line 322?

Reply:
I was thinking of line 352.

Reply:
I see that make sense now. I have moved this in the updated patch.